src/main/java/io/github/moderngamingworlds_mods/woodenutilities/common/recipes/WoodcutterRecipeBuilder.java

```diff
@@ -0,0 +1,87 @@
+package io.github.moderngamingworlds_mods.woodenutilities.common.recipes;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import io.github.moderngamingworlds_mods.woodenutilities.WoodenUtilities;
+import io.github.moderngamingworlds_mods.woodenutilities.common.init.ModRecipes;
+import net.minecraft.data.recipes.FinishedRecipe;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Consumer;
+
+public class WoodcutterRecipeBuilder implements FinishedRecipe {
```

Comment:
That is not how a builder really works. As an example, look here: https://github.com/MelanX/Cucurbita/blob/1.16.x/src/main/java/de/melanx/cucurbita/api/recipe/builders/HollowedPumpkinBuilder.java

Reply:
yeah I know, but in this case I had to rewrite this because intellij corrupted the file, and I wanted to go to bed

Reply:
In that case: just do it on another day.